test/unit/session_test.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const { expect } = require('chai')
+const session = require('../../app/utils/session')
+
+const chargeId = 'foo'
+
+var EMPTY_RESPONSE = { params: {}, body: {}, get: () => null }
+
+var NO_SESSION_GET_RESPONSE = {
+    params: { chargeId: chargeId },
+    body: {},
+    method: 'GET',
+    get: () => null
+}
+
+var NO_SESSION_POST_RESPONSE = {
+    params: {},
+    body: { chargeId: chargeId },
+    method: 'POST',
+    get: () => null
+}
+
+var VALID_GET_RESPONSE = {
+    params: { chargeId: chargeId },
+    body: {},
+    method: 'GET',
+    frontend_state: { ch_foo: true },
+    get: () => null
+}
+
+var VALID_POST_RESPONSE = {
+    params: {},
+    body: { chargeId: chargeId },
+    method: 'POST',
+    frontend_state: { ch_foo: true },
+    get: () => null
+}
+
+describe('session utils ', () => {
+    describe('createChargeIdSessionKey ', () => {
+        it(`should return chargeId with prefix ch_`, function () {
```

Comment:
_very minor_  but string interpolation (``) doesn't need to be used unless there is dynamic content, in this case favour ('')

Reply:
done